README.md

```diff
@@ -1,108 +1,28 @@
 # Todo App Java On Azure
 
-This TodoList app is an Azure Java application. It provides end-to-end CRUD operation to todo list item from front-end AngularJS code. Behind the scene, todo list item data store is [Azure CosmosDB DocumentDB](https://docs.microsoft.com/en-us/azure/cosmos-db/documentdb-introduction). This application uses [Azure CosmosDB DocumentDB Spring Boot Starter](https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-documentdb-spring-boot-starter) and AngularJS to interact with Azure. This sample application provides several deployment options to deploy to Azure, pls see deployment section below. With Azure support in Spring Starters, maven plugins and Eclipse/IntelliJ plugins, Azure Java application development and deployment is effortless now.
-
-
-## TOC
-
-* [Requirements](#requirements)
-* [Create Azure Cosmos DB documentDB](#create-azure-cosmos-db-documentdb)
-* [Configuration](#configuration)
-* [Run it](#run-it)
-* [Contribution](#contribution)
-* Add new features
-    * [Add AAD](https://github.com/Microsoft/todo-app-java-on-azure/tree/aad-start)
-    * [Add KeyVault]()
-* Deployment
-    * [Deploy to Azure Web App for Containers using IntelliJ plugin](./doc/deployment/deploy-to-azure-web-app-using-intelliJ-plugin.md)
-    * [Deploy to Azure Web App for Containers using Eclipse plugin](./doc/deployment/deploy-to-azure-web-app-using-eclipse-plugin.md)
-    * [Deploy to Azure Container Service Kubernetes cluster using Maven plugin](./doc/deployment/deploy-to-azure-container-service-using-maven-plugin.md)
-    * [Deploy to Azure Web App for Containers using Maven plugin](./doc/deployment/deploy-to-azure-web-app-using-maven-plugin.md)
-* [Useful link](#useful-link)
+This TodoList app is an Azure Java application. It provides end-to-end CRUD operation to todo list item from front-end AngularJS code. Behind the scene, todo list item data store is [Azure CosmosDB DocumentDB](https://docs.microsoft.com/en-us/azure/cosmos-db/documentdb-introduction). This application uses [Azure CosmosDB DocumentDB Spring Boot Starter](https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-documentdb-spring-boot-starter) and AngularJS to interact with Azure. In particular, this version of the application shows how to use [Azure Application Insights](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started) to monitor the performance and usage of the application.
```

Comment:
Should we still point to https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started. as it doesn't have instructions on configuration with SpringBoot. Might be customers get mislead? Just brainstorming, if answer is no - where should we point them.

Reply:
Thanks for the reminder - I meant to ask Asir about whether there were plans to improve this documentation to mention SpringBoot.  Also, I will clarify in the ReadMe that this sample shows SpringBoot and that for other examples, refer to the 'getting started' docs.